includes/definitions/alcatel-omnipcx.yaml

```diff
@@ -0,0 +1,13 @@
+os: alcatel-omnipcx
+group: alcatel
+text: 'Alcatel OmniPCX'
+type: server
```

Comment:
I wonder if appliance would better fit here.

Reply:
That makes sense yes. Will be changed.